code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -9,7 +9,7 @@
 	health = 20
 	maxHealth = 20
 	pass_flags = PASSMOB
-
+	status_flags = (CANPUSH | CANSTUN)
```

Comment:
This appears to be a new change?

Reply:
I think this is needed for the medibot to appear stunned later in the file